src/routes/users/leaderboard.ts

```diff
@@ -0,0 +1,30 @@
+import { Route, RouteStore } from 'klasa-dashboard-hooks';
+import ApiRequest from '../../lib/structures/api/ApiRequest';
+import ApiResponse from '../../lib/structures/api/ApiResponse';
+import { ratelimit, iteratorRange, fetchAllLeaderboardEntries } from '../../lib/util/util';
+
+export default class extends Route {
+
+	public constructor(store: RouteStore, file: string[], directory: string) {
+		super(store, file, directory, { name: 'globalLeaderboard', route: 'users/leaderboard' });
+	}
+
+	@ratelimit(2, 2500)
+	public async get(request: ApiRequest, response: ApiResponse) {
+		const limit = 'limit' in request.query ? Number(request.query.limit) : 10;
```

Comment:
```suggestion
		const limit = Number(request.query?.limit || 10);
```

If LGTM already supports optional chaining then we can use it here